depthai_nodes/ml/parsers/yunet.py

```diff
@@ -120,22 +120,26 @@ def setOutputLayerIou(self, iou_output_layer_name: str) -> None:
     def build(
         self,
         head_config: Dict[str, Any],
+        inputs_size: List[List[int]],
     ) -> "YuNetParser":
         """Configures the parser.
 
         @param head_config: The head configuration for the parser.
         @type head_config: Dict[str, Any]
+        @param inputs_size: Model inputs size.
+        @type inputs_size: List[List[int]]
         @return: The parser object with the head configuration set.
         @rtype: YuNetParser
         """
 
         super().build(head_config)
         output_layers = head_config.get("outputs", [])
-        self.input_size = head_config.get("input_size", self.input_size)
-        if len(output_layers) != 3:
+        if len(inputs_size) != 1:
             raise ValueError(
-                f"YuNetParser expects exactly 3 output layers, got {output_layers} layers."
+                f"Only one input supported for LaneDetectionParser, got {len(inputs_size)} inputs."
```

Comment:
This is talking about LaneDetectionParser but is inside YuNet?

Reply:
Fixed in [7b999ac](https://github.com/luxonis/depthai-nodes/pull/126/commits/7b999ac26952c60925c31d1969a99e49a018173c).